modules/tensor_mechanics/test/tests/postprocessors/nodeset_force/nodeset_force.i

```diff
@@ -0,0 +1,104 @@
+[GlobalParams]
+  displacements = 'disp_x disp_y disp_z'
+[]
+
+[Problem]
+  extra_tag_vectors = 'ref'
+[]
+
+[Mesh]
+  [gmg]
+    type = GeneratedMeshGenerator
+    dim = 3
+    nx = 1
+    ny = 1
+    nz = 1
+  []
+[]
+
+[AuxVariables]
+  [react_x]
+  []
+[]
+
+[AuxKernels]
+  [react_x]
+    type = TagVectorAux
+    vector_tag = 'ref'
+    v = 'disp_x'
+    variable = 'react_x'
+  []
+[]
+
+[Postprocessors]
+  [react_x]
+    type = NodesetForce
+    variable = disp_x
+    vector_tag = 'ref'
+    boundary = right
+  []
+[]
+
+[Modules/TensorMechanics/Master]
+  [plane_stress]
```

Comment:
Despite the name of this block, you're getting plane strain here rather than plane stress.